marklogic-data-hub-central/src/main/resources/application.properties

```diff
@@ -7,20 +7,8 @@ spring.thymeleaf.prefix=classpath:/static/
 
 # Data Hub connection properties. These default to the expected values for DHS.
 mlHost=localhost
-mlIsHostLoadBalancer=true
-mlIsProvisionedEnvironment=true
-mlManageScheme=https
-mlManageSimpleSsl=true
-mlAppServicesAuthentication=basic
-mlFinalAuth=basic
-mlJobAuth=basic
-mlStagingAuth=basic
-mlAppServicesSimpleSsl=true
-mlFinalSimpleSsl=true
-mlJobSimpleSsl=true
-mlStagingSimpleSsl=true
-# Need this for getting the version of MarkLogic via the Versions class
-mlAppServicesPort=8010
+hubDhs=true
+hubSsl=true
```

Comment:
I am guessing ,we need 2 separate properties because we aren't sure if HC will need SSL to talk to ML. If we decide that we need SSL, one property should be sufficient ?

Reply:
Exactly - hopefully we only need one soon, just not there yet.